deploy/charts/jetstack-agent/README.md.gotmpl

```diff
@@ -5,54 +5,122 @@
 
 ## Additional Information
 
-The Jetstack secure agent helm chart installs the Kubernetes agent that connects to The TLS Protect For Kubernetes platform.
+The Jetstack Secure agent helm chart installs the Kubernetes agent that connects to the TLS Protect For Kubernetes (TLSPK) platform.
```

Comment:
The TLS Protect for Kubernetes agent Helm chart

Reply:
I think I caught this with my latest commit